crates/sui-framework/packages/dwallet-system/tests/proof_tests.move

```diff
@@ -0,0 +1,42 @@
+#[test_only]
+module dwallet_system::proof_tests {
+    use std::vector;
+    use dwallet::test_scenario;
+    use dwallet::test_scenario::TransactionEffects;
+    use dwallet_system::proof::launch_proof_mpc_flow;
+
+    // <<<<<<<<<<<<<<<<<<<<<<<< Error codes <<<<<<<<<<<<<<<<<<<<<<<<
+    const EWrongEventNumber: u64 = 0;
+    const EWrongFrozenObjectsNum: u64 = 1;
+    const EWrongCreatedObjectsNum: u64 = 2;
+    const EObjectMismatchCreateAndFrozen: u64 = 3;
+    // <<<<<<<<<<<<<<<<<<<<<<<< Error codes <<<<<<<<<<<<<<<<<<<<<<<<
+
+
+    fun set_up(): (address, test_scenario::Scenario) {
+        let sender = @0x1;
+        let scenario = test_scenario::begin(sender);
+        (sender, scenario)
+    }
+
+    #[test]
+    public fun test_launch_proof_mpc_flow_succesfull() {
+        let (sender, scenario) = set_up();
+        test_scenario::next_tx(&mut scenario, sender);
+        {
+            let ctx = test_scenario::ctx(&mut scenario);
+            launch_proof_mpc_flow(ctx);
+        };
```

Comment:
Just curious (I don't know much about `.move`): is there a reason you're putting this in context?

Reply:
IDK honestly, I remember once I didn't do it, the code worked fine but @zeevmoney  asked me to put it in a context. @zeevmoney wdyt

Reply:
It must be done like this in order to use the test_scenario feature which enables better tests.
It keep track of a fake execution context.

Reply:
@erik-3milabs @ItayLevyOfficial